### PR TITLE
glean: 1547273: Display error from GleanDebugActivity when glean isn't initialized

### PR DESCRIPTION
**Pull request by mdboom**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: components/service/glean/src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt [hunk 1]
```diff
@@ -41,6 +41,14 @@ class GleanDebugActivity : Activity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        if (!Glean.isInitialized()) {
```
Comment (Dexterp37) [inline components/service/glean/src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt]: Does this work even if the application is started by the intent?

Comment (mdboom) [reply components/service/glean/src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt]: It seems to.  i.e. I tested by killing the app in the application manager and then running the `adb` command (on a version of `sample-glean` with `Glean.initialize` commented out).

Comment (Dexterp37) [reply components/service/glean/src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt]: I wonder if this is racing with the init from the main activity :\

Comment (mdboom) [reply components/service/glean/src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt]: Well, looking into this sent me down the rabbit hole where I found #2863 which is much higher priority.  Let's get that merged then come back to this.

Comment (mdboom) [reply components/service/glean/src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt]: This seems to work now that #2863 is implemented, so ready for another review.

**Pull request merged**